src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Filter } from '../Filter';
+import { FilterStatus } from '../../types/FilterStatus';
+
+type Props = {
+  filterStatus: string;
```

Comment:
```suggestion
  filterStatus: FilterStatus ;
```
`filterStatus` should have `FilterStatus` type